### DISCUSSION TITLE
how to number figure  like `Figure 1-1` instead of `Figure 1.1`

### DISCUSSION THREAD

**Discussion by Achuan-2** [quarto-dev/quarto-cli#3358] [Q&A]
I have a requirement. When `number-sections: true` is turned on, I want to generate a format like `Figure 1-1` instead of `Figure 1.1`. What should I do for pdf and gfm output?

**Comment by mcanouil** [comment]
I am afraid you can't ...
Maybe a Lua filter can help, but in the cross-reference scope, I am not so sure.
